Week2/nadin.js

```diff
@@ -0,0 +1,181 @@
+'use strict'
+
+//strings
+
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);
+console.log(myString.length); 
+
+const space = " ";
+myString = 'hello' + space + 'this' + space + 'is' + space + 'a' + space + 'difficult' + space + 'to' + space + 'read' + space + 'sentence';
+console.log(myString);
```

Comment:
This works but it is not really effective, take a look again at `replace` and  `split` . These functions work to split up a string. For example `myString.replace(/,/g, ` `)`, where you replace the `,` character with a space.

Reply:
Thanks! That help me well to do it on easy way and also it is shorter. :) 👍